static/css/style.css

```diff
@@ -31,6 +31,10 @@ body {
     height: 100%;
 }
 
+html form.w-100 {
```

Comment:
Does this not happen by default? It's a Bootstrap class, so something else might be blocking this from functioning on your form... 🤔

Reply:
`w-100` does not set `max-width`, just `width` and there is another css rule for `html form` further up that sets the `max-width` so I had to be more specific here.